R/rake_functions.R

```diff
@@ -287,10 +287,10 @@ extract_weights <- function(weights, inds, id) {
 #' weights     <- weight(cons = cons, inds = inds, vars = vars)
 #' weights_int <- integerise(weights)
 #' weights_int
-integerise <- function(weights, method = "trs") {
+integerise <- function(weights, method = "trs", seed = 42) {
```

Comment:
Calling seed as an argument with a default fixes the seed (which can be manually changed).
Without it the seed changes unless it's manually set.